bootloaders/riotboot_serial/Makefile

```diff
@@ -0,0 +1,10 @@
+# Default RIOT bootloader
+APPLICATION = riotboot_serial
+
+# Include riotboot flash partition functionality
+# USEMODULE += riotboot_slot
```

Comment:
I think it's better to remove `riotboot_slot`, right now the bootloader can't possible support `riotboot_slot` since it can only flash `hexfiles` and all riotboot functionality relies on flashing `bin`.

Sure it could initially be bootstrapped with a riotboot_serial + riotboot_slot firmware but no firmware upgrades over serial can be made.

Reply:
Ok, although that's not a limitation of the bootloader but the flasher tool

Reply:
Yes, but will it make sense to use the bootloader in that case?